_articles/building-community.md

```diff
@@ -1,6 +1,6 @@
 ---
 lang: en
-title: Building Welcoming Communities
```

Comment:
Did you mean to remove the title?

Reply:
Me?

Reply:
No @jmeridth

Reply:
@CBID2 if you didn't mean to remove the title, could we please put it back?